site/content/en/docs/tasks/run_jobs.md

```diff
@@ -19,6 +19,8 @@ Make sure the following conditions are met:
 - [Kueue is installed](/docs/installation).
 - The cluster has [quotas configured](/docs/tasks/administer_cluster_quotas).
 
+![Components](/images/components-diagram.svg)
```

Comment:
Add an accompanying text that this is an overview of the concepts that you will interact with in this tutorial.